pandas/io/formats/style.py

```diff
@@ -596,58 +590,101 @@ def format(self, formatter, subset=None, na_rep: Optional[str] = None) -> Styler
 
             .. versionadded:: 1.0.0
 
+        precision : int, optional
+            Floating point precision to use for display purposes, if not determined by
+            the specified ``formatter``.
+
+            .. versionadded:: 1.3.0
+
         Returns
         -------
         self : Styler
 
         Notes
         -----
-        ``formatter`` is either an ``a`` or a dict ``{column name: a}`` where
-        ``a`` is one of
+        This method assigns a formatting function to each cell in the DataFrame. Where
```

Comment:
can you try to make this more clear (i know you are reformatting the existing), but this is not obvious at all.

Reply:
i expanded in the notes and cited them. hope this is ok, if not preferred will try and condense for arg snippet.